lib/util.js

```diff
@@ -871,12 +878,13 @@ exports.getProxyFromEnv = function (isHttps = true) {
   Logger.getInstance().debug(`Util.getProxyEnv: Using ${protocol.toUpperCase()}_PROXY from the environment variable`);
   if (proxyFromEnv.indexOf('://') === -1) {
     Logger.getInstance().info('Util.getProxyEnv: the protocol was missing from the environment proxy. Use the HTTP protocol.');
-    proxyFromEnv = 'http' + '://' + proxyFromEnv;
+    proxyFromEnv = protocol + '://' + proxyFromEnv;
```

Comment:
please, let's use HTTP if the user doesn't define the protocol. See above comment.